dashboard/app/batch_reproexport.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2024 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import (
+	"net/http"
+
+	"google.golang.org/appengine/v2"
+	"google.golang.org/appengine/v2/log"
+)
+
+const exportTimeoutSeconds = 10000 * 2 // upstream has apx 7k reproducers, 1s each max (throttling)
+
+func handleBatchReproExport(w http.ResponseWriter, r *http.Request) {
+	ctx := appengine.NewContext(r)
+	reproExportPath := getConfig(ctx).ReproExportPath
+	if reproExportPath == "" {
+		return
+	}
+	if err := createScriptJob(ctx, "syzkaller", "export-repro",
+		exportReproScript(reproExportPath),
+		exportTimeoutSeconds, nil); err != nil {
+		log.Errorf(ctx, "createScriptJob: %s", err.Error())
+	}
+}
+
+func exportReproScript(archivePath string) string {
+	script := "\n" +
+		// "git clone --depth 1 --branch master --single-branch https://github.com/google/syzkaller\n" +
+		"git clone https://github.com/tarasmadan/syzkaller\n" +
+		"cd syzkaller\n" +
+		"git checkout syz_reprolist_by_namespace\n" +
+		"export CI=1\n" +
+		"./tools/syz-env \"" +
+		"go run ./tools/syz-reprolist/... -namespace upstream; " +
```

Comment:
"upstream" shouldn't be hardcoded here. It does not necessary exist, and it's not the only namespace we want to export from.
You configured the bucket in the config, but "upstream" is also part of config.

Reply:
Accepted. I'll add one more option.

Reply:
> and it's not the only namespace we want to export from
I was sure it is the only for now.

Reply:
Let's better make the ReproExportPath a per-ns option.